src/core/errors/errorreporter.js

```diff
@@ -85,7 +85,7 @@ export default class ErrorReporter {
 
     if (this.sendToServer) {
       const requestConfig = {
-        endpoint: '/v2/accounts/me/answers/errors',
+        endpoint: '/v2/accounts/me/search/errors',
```

Comment:
I'm not sure if anyone uses this anymore, but I suppose it doesn't hurt to update it